.build/tasks/Common.Functions.psm1

```diff
@@ -130,6 +117,45 @@ function Get-ModuleVersion
     return $moduleVersionParts.ModuleVersion
 }
 
+function Get-ModuleVersion
```

Comment:
non-blocking:
Get-BuiltModuleVersion?
I'm just afraid Get-ModuleVersion might be so generic that it conflicts with another command one day...

Reply:
Yes, you are right. Fixing.